core/lib/runtime/enter.c

```diff
@@ -33,7 +33,7 @@
 #include "util/util.h"
 #include "util/message.h"
 #include "util/privilege.h"
-#include "./chroot/chroot.h"
+#include "runtime/chroot.h"
```

Comment:
We certainly can. This pull request is big enough as it stands because of the new elements brought in. I wanna do that in another iteration if possible. Better tracking of similar changes. In other words, I didn't wanna touch the C/H files to much this turn around.